cyclus/simstate.py

```diff
@@ -58,9 +58,9 @@ class SimState(object):
         The initial registry to start the in-memory backend with. Defaults
         is True, which stores all of the tables.
     schema_path : str or None, optional:
-        The path to the cyclus master schema.
+        The path to the cyclus main schema.
```

Comment:
Same comment about the schema vocabulary.

Reply:
ditto